xmodule/util/builtin_assets.py

```diff
@@ -12,6 +12,27 @@
 from openedx.core.djangoapps.theming.helpers_static import get_static_file_url
 
 
+def add_css_to_fragment(fragment, css_relative_path):
+    """
+    Given a css path relative to xmodule/assets, add a compiled CSS URL to the fragment.
+
+    Raises:
+    * ValueError if {css_relative_path} is absolute or does not end in '.css'.
+    * FileNotFoundError if edx-platform/xmodule/assets/{css_relative_path} is missing.
+    """
+    if not isinstance(css_relative_path, Path):
+        css_relative_path = Path(css_relative_path)
+    if css_relative_path.is_absolute():
+        raise ValueError(f"css_file_name should be relative; is absolute: {css_relative_path}")
+    if css_relative_path.suffix != '.css':
+        raise ValueError(f"css_file_name should be .css file; is: {css_relative_path}")
+    css_absolute_path = Path(settings.REPO_ROOT) / "xmodule" / "assets" / css_relative_path
+    if not css_absolute_path.is_file():
+        raise FileNotFoundError(f"css file not found: {css_absolute_path}")
+    # fragment.add_css_url(str(css_absolute_path))
+    fragment.add_css(css_absolute_path.read_text(encoding='utf-8'))
```

Comment:
@kdmccormick I have changed css url addition to css content addition as css url is not accessible on browser and css file needs to add in the static files.

Please guide what should be the way forward of implementation?